tstest/integration/vms/vms_test.go

```diff
@@ -80,6 +86,40 @@ func (d *Distro) InstallPre() string {
 	return ""
 }
 
+// fetchFromS3 fetches a distribution image from Amazon S3 or returns false if it
```

Comment:
boolean func doc style in Go is "reports whether" ...

"and reports whether it succeeded".

But this func is weird:

* it returns a bool, saying it's about permission
* but two false returns are not about permission
* it also t.Fatals on failure (but why not when NewSession fails?)

Reply:
I think 91bdf46a gives enough context to explain what's going on. Should I use a named return value to be more explicit?